src/basic/audit-util.c

```diff
@@ -92,8 +92,11 @@ bool use_audit(void) {
                 int fd;
 
                 fd = socket(AF_NETLINK, SOCK_RAW|SOCK_CLOEXEC|SOCK_NONBLOCK, NETLINK_AUDIT);
-                if (fd < 0)
-                        cached_use = errno != EAFNOSUPPORT && errno != EPROTONOSUPPORT;
+                if (fd < 0) {
+                        cached_use = errno != EAFNOSUPPORT && errno != EPROTONOSUPPORT && errno != EPERM;
```

Comment:
not that it matters, but we have this pretty macro IN_SET() for cases like this:

`cached_use = !IN_SET(errno, EAFNOSUPPORT, EAPROTONOSUPPORT, EPERM)`